spec/spec.md

```diff
@@ -1698,13 +1698,19 @@ Schema: [`Demographics.schema.json`](Demographics.schema.json)
 
 ### Example
     [{ 
-        "code" : "as-fert",
+        "id" : "as-fert",
         "name": "Age-specific fertility",
+        "variants" : ["low", "medium"],
```

Comment:
hmm I thought I took this out. I think we remove all references to variants for now